src/tests/ftest/util/command_utils_base.py

```diff
@@ -789,6 +799,76 @@ def _get_new(self):
         return TransportCredentials(self.namespace, self.title, self._log_dir)
 
 
+class TelemetryCredentials(YamlParameters):
+    """Telemetry credentials listing certificates for secure communication."""
+
+    def __init__(self, namespace, title, log_dir):
+        """Initialize a TelemetryConfig object.
```

Comment:
```suggestion
        """Initialize a TelemetryCredentials object.
```

Reply:
Will update

Reply:
This comment just needs to match whatever the class name is